web/src/actions/apd.js

```diff
@@ -154,19 +154,18 @@ export const notifyNetError = (action, error) => {
 };
 
 export const saveApd = ({ serialize = toAPI } = {}) => (dispatch, state) => {
-  dispatch(requestSave());
-
   const {
     apd: { data: updatedApd },
     activities,
     dirty
   } = state();
 
   if (!dirty.dirty) {
-    dispatch(notify('Save successful!'));
     return Promise.resolve();
   }
 
+  dispatch(requestSave());
+
```

Comment:
Switched this up so it only sends the save request action if it actually does the network request.